packages/Breadcrumbs/Breadcrumbs.md

```diff
@@ -102,3 +102,27 @@ const BreadcrumbNavigation = ({ location }) => {
   return <Breadcrumbs reactRouterLinkComponent={ReactRouterLink}>{breadcrumbItems}</Breadcrumbs>
 }
 ```
+
+#### Combining React Router Links with standard anchor links
+
+If you need to combine both `react-router-dom` and TDS `Link` components, you can add a `reactRouterLinkComponent` prop to `Breadcrumbs.Item` where needed as show below.
+
+```jsx static
+// BreadcrumbNavigation.jsx
+import { Link as ReactRouterLink } from 'react-router-dom'
+import Breadcrumbs from '@tds/core-breadcrumbs'
+
+const Item = Breadcrumbs.Item
```

Comment:
For simplicity and easy copy pasting's sake, let's avoid assigning `Breadcrumbs.Item` to `Item` and then using `Item` in the code example.